src/test/groovy/org/c02e/jpgpj/EncryptorSpec.groovy

```diff
@@ -763,6 +771,7 @@ hQEMAyne546XDHBhAQ...
         meta.verified
     }
 
+    @Ignore
```

Comment:
I couldn't make this work when I run this with `./gradlew build`. 
It gives me the following error: 
```
org.bouncycastle.openpgp.PGPException: exception constructing public key
	at org.c02e.jpgpj.Encryptor.encrypt(Encryptor.java:1130)
	at org.c02e.jpgpj.Encryptor.prepareCiphertextOutputStream(Encryptor.java:974)
	at org.c02e.jpgpj.Encryptor.encrypt(Encryptor.java:857)
	at org.c02e.jpgpj.Encryptor.encrypt(Encryptor.java:826)
	at org.c02e.jpgpj.EncryptorSpec.encrypt and sign with no usage flags(EncryptorSpec.groovy:781)
Caused by: java.security.spec.InvalidParameterSpecException: Not a supported curve: java.security.spec.ECGenParameterSpec@5b0e7c05
	at java.base/sun.security.util.ECParameters.engineInit(ECParameters.java:130)
	at java.base/java.security.AlgorithmParameters.init(AlgorithmParameters.java:294)
	... 5 more
```
Could you please take a look

Reply:
This is just an artifact of some peculiarities with groovy compilation and class loading (it fails here and not elsewhere because the JDK's default JCA implementation can handle the other test public keys, but not this test key using the P-256 curve). I believe right now it fails only when this class is recompiled and run in the same process (and not when compiled and run separately); and it will go away if a `BouncyCastleProvider` instance is set directly on the `JcaContextHelper.securityProvider` property (instead of indirectly via the JCA apparatus of `Security.addProvider()` / `Security.getProvider()`).